src/screens/SingleChoiceExerciseScreen.tsx

```diff
@@ -1,8 +1,40 @@
-import { Text } from 'react-native'
 import React from 'react'
+import { SingleChoice } from '../components/SingleChoice'
+import { ISingleChoiceListItemProps } from '../components/SingleChoiceListItem'
 
 const SingleChoiceExerciseScreen = () => {
-  return <Text>Single Choice</Text>
+  const answerOptions: ISingleChoiceListItemProps[] = [
+    {
+      article: 'das',
+      word: 'Schraube',
+      correct: false,
+      selected: false,
+      addOpacity: true
```

Comment:
Which items get opacity? I don't understand this :see_no_evil:

Reply:
Also, why are items selected automatically already? I think items should only be selected once I click on them.

Reply:
That is just mockdata to show that the styling of selected items and items that have addOpacity set to true is correct. The functionality of setting the props to the right values should be implemented in a later issue.